action.yml

```diff
@@ -266,7 +266,7 @@ runs:
       working-directory: ${{ inputs.path }}
       run: |
         yq -n '.shopware.api.jwt_key.use_app_secret = true' | tee config/packages/zz-shopware.yaml
-      if: ${{ hashFiles(format('{0}/src/Core/Framework/Api/OAuth/JWTConfigurationFactory.php', (inputs.path || '.' ))) != '' }}
+      if: ${{ hashFiles(format('{0}/src/Core/Maintenance/System/Service/JwtCertificateGenerator.php', (inputs.path || '.' ))) != '' }}
```

Comment:
this will break 6.7 as that config is gone. you can delete the complete step

Reply:
it seems to be not working as expected or do we somehow need to release it?

Just removing this line would break all 6.6 usage of this action right?

what I wanted to achieve was only executed this step when that referenced class is there

Reply:
https://github.com/tinect/TinectRedirects/actions/runs/12889918750/job/35938294628#step:2:858 setup broke here already